spring-kafka-test/src/main/java/org/springframework/kafka/test/context/EmbeddedKafka.java

```diff
@@ -85,6 +85,8 @@
 	int partitions() default 2;
 
 	/**
+	 * Topics that should be created
+	 * Topics may contain property placeholders, e.g. {@code topics = "${kafka.topic.one:topicOne}"}
```

Comment:
I wasn't sure if javadoc counted as a cosmetic change, so I didn't add an `@author` tag. Let me know if that needs to be added.

Reply:
That’s change anyway, so, please, add your name to the `@author`. Thanks

Reply:
Added. Thanks @artembilan